### ISSUE TITLE
DropDownSettingsTile leading Icon

### ISSUE THREAD

**Issue by azazadev** [GAM3RG33K/flutter_settings_screens#30]
Hi,
It is possible to have leading Icon for DropDownSettingsTile ?

**Comment by zuzi-m** [comment]
Yes it is - I just opened a PR #62 with that enhancement - it adds the option to add a `leading` widget to `DropDownSettingsTile` and some others that didn't support it before
